devtools/src/arcs-hcr-list.js

```diff
@@ -0,0 +1,305 @@
+import './common/object-explorer.js';
+
+class ArcsHcrList extends MessengerMixin(PolymerElement) {
+  static get template() {
+    return html`
+    <style include="shared-styles">
+      :host {
+        line-height: 24px;
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        display: flex;
+        flex-direction: column;
+      }
+      header {
+        flex-grow: 0;
+      }
+      [main] {
+        flex-grow: 1;
+        overflow-y: auto;
+      }
+      .title {
+        padding: 0 8px;
+        height: 26px;
+        border-bottom: 1px solid var(--mid-gray);
+        background-color: var(--light-gray);
+      }
+      .refresh {
+        -webkit-mask-position: -84px 48px;
+        cursor: pointer;
+        transition: transform .5s;
+        vertical-align: middle;
+      }
+      :host([loading]) .refresh {
+        transform: rotate(1turn);
+      }
+      .content {
+        border-bottom: 1px solid var(--mid-gray);
+        display: flex;
+        flex-direction: column;
+      }
+      object-explorer {
+        margin: 2px 4px;
+      }
+      [name]:not(:empty) {
+        color: var(--devtools-purple);
+        margin-right: 1ch;
+      }
+      [id] {
+        color: var(--devtools-red);
+      }
+      object-explorer[find]:not([found]) {
+        display: none;
+      }
+
+      .switch {
+        position: relative;
+        display: inline-block;
+        width: 30px;
+        height: 17px;
+      }
+      .slider {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background-color: #ccc;
+        border-radius: 34px;
+      }
+      .slider:before {
+        position: absolute;
+        content: "";
+        height: 13px;
+        width: 13px;
+        left: 2px;
+        bottom: 2px;
+        background-color: white;
+        -webkit-transition: .4s;
+        border-radius: 50%;
+      }
+      input:checked + .slider {
+        background-color: #AFF196;
+      }
+      input:checked + .slider:before {
+        -webkit-transform: translateX(13px);
+      }
+      .toggleicon .toggletext {
+        width: 120px;
+        color: #838783;
+        font-size: 10px;
+        position: absolute;
+        top: 8px;
+        left: 35px;
+      }
+      .toggleicon {
+        position: relative;
+        display: inline-block;
+      }
+    </style>
+    <header class="header">
+      <div section>
+        <span class="devtools-icon refresh" on-click="_forceReload"></span>
+        <div divider></div>
+        <div class="toggleicon">
+          <label class="switch">
+            <input type="checkbox" on-click="_loadServer" id="toggle">
+            <span class="slider round"></span>
+          </label>
+          <span class="toggletext">
+            Hot Code Reload
+          </span>
+        </div>
+      </div>
+    </header>
+
+    <div main>
+        <div class="title">Reloadable Particles</div>
```

Comment:
2 less spaces of indentation

Reply:
Indentation fixed! Also removed all the other things!